lib/msf/core/exploit/powershell.rb

```diff
@@ -9,12 +9,13 @@ def initialize(info = {})
       [
         OptBool.new('Powershell::persist', [true, 'Run the payload in a loop', false]),
         OptInt.new('Powershell::prepend_sleep', [false, 'Prepend seconds of sleep']),
+        OptBool.new('Powershell::prepend_protections_bypass', [true, 'Prepend AMSI/SBL bypass', true]),
         OptBool.new('Powershell::strip_comments', [true, 'Strip comments', true]),
         OptBool.new('Powershell::strip_whitespace', [true, 'Strip whitespace', false]),
         OptBool.new('Powershell::sub_vars', [true, 'Substitute variable names', false]),
         OptBool.new('Powershell::sub_funcs', [true, 'Substitute function names', false]),
-        OptBool.new('Powershell::exec_in_place', [true, 'Produce PSH without executable wrapper', false]),
-        OptBool.new('Powershell::remove_comspec', [true, 'Produce script calling powershell directly', false]),
+        OptBool.new('Powershell::exec_in_place', [true, 'Produce PSH without executable wrapper', true]),
```

Comment:
@phra surely this will have side effects in other modules?

Reply:
And surely it was already true in web_delivery? https://github.com/rapid7/metasploit-framework/blob/master/modules/exploits/multi/script/web_delivery.rb#L170